upki/src/fetch.rs

```diff
@@ -237,8 +237,12 @@ impl PlanStep {
             }
             Self::SaveManifest { manifest, local } => {
                 debug!("saving manifest");
-                let file = File::create(local).wrap_err("failed to create manifest file")?;
+
+                let local_temp = local.with_extension(".tmp");
```

Comment:
Might be worth bringing in the tempfile which has some good utilities for this kind of thing?